### PR TITLE
feat: People Panel

### PR DESCRIPTION
**Pull request by cyaiox**
This PR adds the right people panel

image

image

image

### CODE REVIEW

File: src/components/VideoConference/ControlBar/PeoplePanelToggleButton/PeoplePanelToggleButton.module.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.container {
+  /* position: absolute; */
+}
```
Comment (kevinszuchet) [inline src/components/VideoConference/ControlBar/PeoplePanelToggleButton/PeoplePanelToggleButton.module.css]: Can we remove this rule?

File: src/components/VideoConference/RightPanel/Chat/ChatEntry/ChatEntry.tsx [hunk 2]
```diff
@@ -37,7 +37,7 @@ export function ChatEntry({ entry, hideName = false, hideTimestamp = false, mess
   }, [entry.message, messageFormatter])
   const time = new Date(entry.timestamp)
   const locale = navigator ? navigator.language : 'en-US'
-
+  console.log(entry.from)
```
Comment (kevinszuchet) [inline src/components/VideoConference/RightPanel/Chat/ChatEntry/ChatEntry.tsx]: Is this log removable?

File: src/components/VideoConference/RightPanel/PeoplePanel/PeoplePanel.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import React, { useCallback } from 'react'
+import classNames from 'classnames'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { Button, Header, Profile } from 'decentraland-ui'
+import { useLayoutContext } from '../../../../hooks/useLayoutContext'
+import type { Props } from './PeoplePanel.types'
+import styles from './PeoplePanel.module.css'
+
+// import { Profile } from 'decentraland-dapps/dist/modules/profile/types'
```
Comment (kevinszuchet) [inline src/components/VideoConference/RightPanel/PeoplePanel/PeoplePanel.tsx]: Can we remove this import?

Comment (cyaiox) [reply src/components/VideoConference/RightPanel/PeoplePanel/PeoplePanel.tsx]: sure! removed

File: src/components/VideoConference/ControlBar/PeoplePanelToggleButton/PeoplePanelToggleButton.module.css (hunk 1 above)
Comment (cyaiox) [reply src/components/VideoConference/ControlBar/PeoplePanelToggleButton/PeoplePanelToggleButton.module.css]: sure! removed

**Review by kevinszuchet — APPROVED** [review]

**Pull request merged**